modules/portworx-data-services/examples/simple/variables.tf

```diff
@@ -0,0 +1,36 @@
+# TEMPLATE: This file was automatically generated with `generate_addon_structure.sh`
+# TEMPLATE: and should be modified as necessary
+# TEMPLATE:
+# TEMPLATE: All variables must have a description and should declare their type.
+# TEMPLATE: Set defaults whenever possible but do not set defaults for required properties.
+# TEMPLATE: Declare all variables in this file, sprawling declarations are difficult to identify.
+# TEMPLATE:
+# TEMPLATE: https://www.terraform.io/docs/language/values/variables.html
+# TEMPLATE: https://www.terraform.io/docs/language/expressions/types.html
+#
+variable "pds_account_id" {
+  type        = string
+  description = "Account ID used to create token for API calls"
+  default     = "db4652ee-8937-47b2-952d-3b883fd2cb33"
```

Comment:
```suggestion
```

Reply:
As discussed, this is a service account internally used by PDS to add the target cluster. All clients has to use this account id to get auth token, which should be used in helm install phase.